usaspending_api/recipient/v2/views/recipients.py

```diff
@@ -225,55 +232,84 @@ def extract_business_categories(recipient_name, recipient_duns, recipient_hash):
     return sorted(business_categories)
```

Comment:
Can you refactor `extract_business_categories()` to retrieve the business categories from ES instead of `UniversalTransactionView`?

Reply:
Request too large for this PR. It runs into the Elasticsearch limitation of sorting more than 10K entries across all shards.